extensions/core/interfaces/calendar/interface.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+	<div v-bind:class="{ inactive: readonly }">
+		<flat-pickr class="form-control"
+                :config="configs"
+                v-model="value"
+                @input="$emit('input', $event)">
+    </flat-pickr>
+  </div>
+</template>
+
+<script>
+import flatPickr from 'vue-flatpickr-component';
+// Need to add base css for flatpickr
+import 'flatpickr/dist/flatpickr.css';
+// import 'flatpickr/dist/themes/material_red.css';
+import mixin from '../../../mixins/interface';
+
+export default {
+	name: 'interface-calendar',
+	computed: {
+		configs() {
+			return {
+				inline: true,
+				minDate: this.options.min,
+				maxDate: this.options.max,
+				// mode: "range"
+			}
+		},
+	},
+	mixins: [mixin],
+	components: {
+    flatPickr
+  },
+}
+</script>
+
+<style lang="scss">
```

Comment:
The style [needs to be `scoped`](https://vue-loader.vuejs.org/guide/scoped-css.html), otherwise (generic) classes can and will cause problems later on. For example, we don't want this `max-width` on all components that use a `.inactive` class.